feat: official resources section (#967)
italia/eidas-it-wallet-docs: docs/it/official-resources.rst

Risorse Ufficiali
-----------------

Questa sezione contiene asset, risorse e artefatti che non possono essere distribuiti sotto forma di documentazione tecnica ma devono essere forniti nel loro formato digitale nativo. Questi includono file sorgente in formati come SVG, immagini raster, repository di codice, file di configurazione e altri asset digitali essenziali per la progettazione e l'implementazione delle Soluzioni Tecniche del Sistema IT-Wallet.
Qualsiasi risorsa tecnica aggiuntiva, repository e documentazione aggiuntiva saranno rese disponibili in questa sezione nelle versioni successive delle presenti specifiche tecniche.

Sito Ufficiale
^^^^^^^^^^^^^^

> Il sito ufficiale del Sistema IT-Wallet è disponibile all'indirizzo seguente. Potrebbe non essere accessibile se in fase di rilascio o in corso di revisione o in aggiornamento:

Reviewer: ```suggestion
Il sito ufficiale del Sistema IT-Wallet è disponibile all'indirizzo seguente:
```

- **Sito Ufficiale IT-Wallet**: `https://www.wallet.gov.it <https://www.wallet.gov.it>`_

Risorse Grafiche
^^^^^^^^^^^^^^^^

Di seguito sono elencate le risorse grafiche ufficiali, disponibili per il download nella cartella `official_resources/ <https://github.com/italia/eidas-it-wallet-docs/tree/versione-corrente/official_resources>`_ di questo repository:

- **Logo IT-Wallet**: Logo ufficiale del Sistema IT-Wallet (formato SVG) - `Scarica il logo <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/en/images/svg/logoIT-Wallet.svg>`_
- **Pittogramma IT-Wallet**: Pittogramma ufficiale del Sistema IT-Wallet - `Scarica il pittogramma <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/symbol-IT-Wallet.svg>`_
- **IT-Wallet ID**: Elemento grafico ufficiale che assicura l’identificazione e rappresentazione univoca del PID tra tutte le Soluzioni Wallet (formato SVG) - `Scarica IT-Wallet ID <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/IT-Wallet-ID.svg>`_

> - **Trust Mark**: Elemento grafico ufficiale per l'attestazione dell'appartenenza al Sistema IT-Wallet - Consulta la sezione :ref:`brand-identity:Trust Mark`;

Reviewer: @peppelinux we decided to provide the Trust Mark as a single .svg in double language (ITA, ENG). The Official Resources folder should provide that material

Reviewer: Also, the folder containing the resources should be \official-resources, not \images

Author: done b074c43c4338fac8cc41a3a89adc016412328ad7

Risorse Ufficiali
-----------------

> Questa sezione contiene tutti i riferimenti utili per la progettazione e l'implementazione del Sistema IT-Wallet, inclusi repository, loghi, risorse grafiche e documentazione aggiuntiva.

Reviewer: I'd move the final section up here, replacing this text
```suggestion
Questa sezione contiene asset, risorse e artefatti che non possono essere distribuiti sotto forma di documentazione tecnica ma devono essere forniti nel loro formato digitale nativo. Questi includono file sorgente in formati come SVG, immagini raster, repository di codice, file di configurazione e altri asset digitali essenziali per la progettazione e l'implementazione delle Soluzioni Tecniche del Sistema IT-Wallet.
Qualsiasi risorsa tecnica aggiuntiva, repository e documentazione aggiuntiva saranno rese disponibili in questa sezione nelle versioni successive delle presenti specifiche tecniche.
```

Sito Ufficiale
^^^^^^^^^^^^^^

> Il sito ufficiale del Sistema IT-Wallet è disponibile all'indirizzo seguente. Qualora non fosse immediatamente accessibile, sarà reso disponibile quanto prima:

Reviewer: ```suggestion
Il sito ufficiale del Sistema IT-Wallet è all'indirizzo seguente:
```

- **Sito Ufficiale IT-Wallet**: `https://www.wallet.gov.it <https://www.wallet.gov.it>`_

Risorse Grafiche
^^^^^^^^^^^^^^^^

> Le seguenti risorse grafiche ufficiali sono disponibili nella cartella `official_resources/ <https://github.com/italia/eidas-it-wallet-docs/tree/versione-corrente/official_resources>`_ di questo repository e possono essere scaricate direttamente da GitHub:

Reviewer: ```suggestion
Di seguito sono elencate le risorse grafiche ufficiali, disponibili per il download nella cartella `official_resources/ <https://github.com/italia/eidas-it-wallet-docs/tree/versione-corrente/official_resources>`_ di questo repository:
```

- **IT-Wallet ID**: Elemento grafico ufficiale per l'identificazione del Sistema IT-Wallet - `Scarica IT-Wallet ID <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/IT-Wallet-ID.svg>`_
- **Logo IT-Wallet**: Logo ufficiale del Sistema IT-Wallet (formato SVG) - `Scarica il logo <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/en/images/svg/logoIT-Wallet.svg>`_
- **Simbolo IT-Wallet**: Pittogramma ufficiale del Sistema IT-Wallet - `Scarica il simbolo <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/symbol-IT-Wallet.svg>`_
- **Trust Mark**: Elemento grafico ufficiale per l'attestazione dell'appartenenza al Sistema IT-Wallet (versioni ENG e ITA) - `Scarica Trust Mark ENG <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/en/images/svg/trustmark-ENG.svg>`_, `Scarica Trust Mark ITA <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/trustmark-ITA.svg>`_

> - **Authentication Button**: Pulsante ufficiale per l'autenticazione tramite IT-Wallet - `Scarica Authentication Button <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/Authentication-button.svg>`_

Reviewer: - I'd move on top Logo and symbol
- the trust mark is not available as a single svg (for now)
In addition, CTAs to download should link to the files available in the Official Resources folder instead of the images folder 

```suggestion
- **Logo IT-Wallet**: Logo ufficiale del Sistema IT-Wallet (formato SVG) - `Scarica il logo <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/en/images/svg/logoIT-Wallet.svg>`_
- **Pittogramma IT-Wallet**: Pittogramma ufficiale del Sistema IT-Wallet - `Scarica il pittogramma <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/symbol-IT-Wallet.svg>`_
- **IT-Wallet ID**: Elemento grafico ufficiale che assicura l’identificazione e rappresentazione univoca del PID tra tutte le Soluzioni Wallet (formato SVG) - `Scarica IT-Wallet ID <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/IT-Wallet-ID.svg>`_
- **Trust Mark**: Elemento grafico ufficiale per l'attestazione dell'appartenenza al Sistema IT-Wallet - Consulta la sezione :ref:`brand-identity:Trust Mark`;
- **Authentication Button**: Pulsante ufficiale per l'autenticazione tramite IT-Wallet - `Scarica Authentication Button <https://raw.githubusercontent.com/italia/eidas-it-wallet-docs/versione-corrente/docs/it/images/svg/Authentication-button.svg>`_
```

I seguenti componenti HTML ufficiali sono disponibili nella cartella `official_resources/ <https://github.com/italia/eidas-it-wallet-docs/tree/versione-corrente/official_resources>`_ di questo repository:

- **Authentication Button Component**: Componente HTML/CSS ufficiale per l'autenticazione tramite IT-Wallet, basato sul framework `dev-kit-italia <https://github.com/italia/dev-kit-italia/tree/main/packages/button>`_ - `Vedi il componente <https://github.com/italia/eidas-it-wallet-docs/tree/versione-corrente/official_resources/authentication-button>`_ - `Demo su GitHub Pages <https://italia.github.io/eidas-it-wallet-docs/official_resources/authentication-button/authentication-button.html>`_

Repository e Risorse Aggiuntive
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Questa sezione contiene asset, risorse e artefatti che non possono essere distribuiti sotto forma di documentazione tecnica ma devono essere forniti nel loro formato digitale nativo. Questi includono file sorgente in formati come SVG, immagini raster, repository di codice, file di configurazione e altri asset digitali essenziali per la progettazione e l'implementazione delle Soluzioni Tecniche del Sistema IT-Wallet.

> Qualsiasi risorsa tecnica aggiuntiva, repository e documentazione aggiuntiva saranno rese disponibili in questa sezione nelle versioni successive delle presenti specifiche tecniche.

Reviewer: I'd move this part on top, replacing the text below "Risorse Ufficiali"

```suggestion
```

Brand Manual
^^^^^^^^^^^^

> Il Brand Manual contenente le linee guida complete per l'utilizzo delle risorse grafiche e dell'identità visiva del Sistema IT-Wallet sarà prossimamente disponibile sul :ref:`sito ufficiale <official-resources:Sito Ufficiale>` e nella sezione Risorse Ufficiali di queste specifiche tecniche.

Reviewer: ```suggestion
Il Brand Manual contenente le indicazioni complete per l'utilizzo delle risorse grafiche e dell'identità visiva del Sistema IT-Wallet sarà prossimamente disponibile sul :ref:`sito ufficiale <official-resources:Sito Ufficiale>`.
```
